content/pt-br/chaos-engineering.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Engenharia do Caos
+status: Completed
+category: conceito
+tags: ["metodologia", "", ""]
+---
+
+## O que é
+
+Engenharia do Caos ou EC é a disciplina de experimentação em um [sistema distribuído](/distributed-systems/) em produção para aumentar a confiança na capacidade do sistema para resistir a condições turbulentas e inesperadas.
+
+## Problema relacionado
+
+As práticas [SRE](/pt-br/site-reliability-engineering/) e [DevOps](/pt-br/devops/) focam-se em técnicas para aumentar a resiliência e confiabilidade do produto.
+A capacidade de um sistema para tolerar falhas enquanto garante um serviço de qualidade é geralmente um requisito de desenvolvimento de software.
+Existem vários aspetos envolvidos que podem levar a interrupções da aplicação, como infraestrutura, plataforma ou outras partes móveis de uma aplicação com base em ([micro serviços](/microservices/)).
+O *Deploy* frequente de novas funcionalidades para ambientes de produção podem resultar em alta probabilidade de indisponibilidade e incidentes críticos — com consequências consideráveis para o negócio.
+
+## Como isso ajuda
+
+Engenharia do Caos é uma técnica para satisfazer os requisitos de resiliência.
+É utilizado para assegurar resiliência contra falhas na infraestrutura, plataforma e falhas aplicacionais.
```

Comment:
I think here we can use `utilizada` instead of `utilizado` since it refers to `Engenharia do Caos`